### PR TITLE
Added Assignment-1 to the week-1 folder

### PR DESCRIPTION
**Pull request by Dibya12345**
This is my first assignment for the ui part 

question no --> 1.2

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[0820/113142.843:ERROR:icu_util.cc(252)] Couldn't mmap icu data file
```
Comment (subhadip-pesto) [inline debug.log]: What is the need of this file?

File: Week-1/question(1.2)/style.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    position: fixed;
+}
+.header-container{
+    padding: 23px 0px 5px 70px;
+    display: flex;
+}
+.header-image{
+    background-image: url("https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg");
+    background-size: 14em;
+    height: 100vh;
```
Comment (subhadip-pesto) [inline Week-1/question(1.2)/style.css]: why 100vh?

File: Week-1/question(1.2)/style.css [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    width: 100vh;
```
Comment (subhadip-pesto) [inline Week-1/question(1.2)/style.css]: and why for width, you have used vh?

File: Week-1/question(1.2)/style.css [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+}
+.footer-container{
+    padding: 15px 0px 5px 10px;
+    display: flex;
+}
+.footer-image{
+    background-image: url("https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg");
+    background-size: 4em;
+    height: 100vh;
+    width: 100vh;
```
Comment (subhadip-pesto) [inline Week-1/question(1.2)/style.css]: why height and width 100vh?

File: debug.log (hunk 1 above)
Comment (Dibya12345) [reply debug.log]: This one is from the vscode

File: Week-1/question(1.2)/style.css (hunk 4 above)
Comment (Dibya12345) [reply Week-1/question(1.2)/style.css]: I resolved the issue

File: debug.log (hunk 1 above)
Comment (subhadip-pesto) [reply debug.log]: don't commit any unnecessary file, please remove it.

**Review by subhadip-pesto — APPROVED** [review]

**Pull request merged**